Basic-Car-Maintenance/Shared/Dashboard/Views/VehicleSelectionView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  VehicleSelectionView.swift
+//  Basic-Car-Maintenance
+//
+//  https://github.com/mikaelacaron/Basic-Car-Maintenance
+//  See LICENSE for license information.
+//
+
+import SwiftUI
+
+struct VehicleSelectionView: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var selectedVehicle: Vehicle?
+    var vehicles: [Vehicle]
+
+    var body: some View {
+        NavigationView {
+            List {
+                Button(action: {
+                    selectedVehicle = nil
+                    dismiss()
+                }, label: {
+                    Text("All Vehicles")
+                        .fontWeight(.bold)
+                        .foregroundColor(.blue)
+                })
+                
+                ForEach(vehicles, id: \.id) { vehicle in
```

Comment:
you don't need `id: \.id` because Vehicle is `Identifiable`

```suggestion
                ForEach(vehicles) { vehicle in
```